tests/test_utils_regression.py

```diff
@@ -0,0 +1,139 @@
+import unittest
+import os
+import json
+import pandas as pd
+import numpy as np
+from sklearn.linear_model import LinearRegression, LogisticRegression
+
+from rotten_tomatoes.utils.regression import RegressionAnalysis, CorrelationAnalysis
+
+class TestRegressionAnalysis(unittest.TestCase):
+    """
+    Class used to test the rotten_tomatoes.utils.regression RegressionAnalysis
+
+    """
+    def test_regressionanalysis_create_linear(self):
```

Comment:
I think all of these functions are going to need docstrings?

Reply:
Updated